src/main/kotlin/gg/essential/essential-loader.gradle.kts

```diff
@@ -0,0 +1,64 @@
+package gg.essential
+
+import gg.essential.gradle.multiversion.Platform
+import gg.essential.gradle.util.RelocationTransform.Companion.registerRelocationAttribute
+
+
+plugins {
+    id("gg.essential.loom")
+}
+
+val platform = Platform.of(project)
+
+val essentialLoader: Configuration by configurations.creating
+
+when {
+    platform.isLegacyForge -> {
+        dependencies {
+            "runtimeOnly"(essentialLoader("gg.essential:loader-launchwrapper:1.1.3")!!)
+        }
+        tasks.named<Jar>("jar") {
+            dependsOn(essentialLoader)
+            from({ zipTree(essentialLoader.singleFile) })
+        }
```

Comment:
Also need to set the TweakClass manifest entry, otherwise Forge can't know that it should be loaded.